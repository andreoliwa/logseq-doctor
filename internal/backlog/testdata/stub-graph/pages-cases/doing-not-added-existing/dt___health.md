- ((67c490ad-f00e-480f-bead-0dcab66c3173))
- The other task in #health is DOING and shouldn't be added here
