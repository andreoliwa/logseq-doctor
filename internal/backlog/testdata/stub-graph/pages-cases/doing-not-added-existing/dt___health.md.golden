- New tasks [[inbox]]
	- ((67c48ea4-92cd-4b27-8202-ec1f4fe4ec59))
- ((67c490ad-f00e-480f-bead-0dcab66c3173))
- The other task in #health is DOING and shouldn't be added here
